bulk_query_gen/influxdb/influx_iot_common.go

```diff
@@ -76,3 +76,28 @@ func (d *InfluxIot) averageTemperatureDayByHourNHomes(qi bulkQuerygen.Query, nHo
 	q := qi.(*bulkQuerygen.HTTPQuery)
 	d.getHttpQuery(humanLabel, interval.StartString(), query, q)
 }
+
+func (d *InfluxIot) IotAggregateKeep(qi bulkQuerygen.Query) {
+	interval := d.AllInterval.RandWindow(7 * 24 * time.Hour)
+
+	var query string
+	if d.language == InfluxQL {
+		query = fmt.Sprintf(`SELECT mean("co2_level") as "mean_value" FROM "air_quality_room" WHERE time > '%s' AND time < '%s' AND "room_id"='4' GROUP BY time(5m) FILL(null)`,
+			interval.StartString(),
+			interval.EndString())
+	} else {
+		query = fmt.Sprintf(`from(bucket: "%s") `+
+			`|> range(start: %s, stop: %s) `+
+			`|> filter(fn: (r) => r._measurement == "co2_level" and r._field == "air_quality_room")`+
```

Comment:
I think this is flipped: when I run the IoT data generation locally, I get a measurement of `air_quality_room` with a field of `co2_level`. I see the same in your InfluxQL query above has the same